mail_template_attachment_i18n/models/ir_attachment_language.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2018 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class IrAttachmentLanguage(models.Model):
+    _name = "ir.attachment.language"
+
```

Comment:
Add _description attribute, it now raises a warning in the logging
```suggestion
    _description = 'Attachment'
```